.github/actions/spell-check/expect.txt

```diff
@@ -721,6 +721,7 @@ libsystemd
 libtdsodbc
 libyaml
 libzmq
+lightningstream
```

Comment:
should this be here?

Reply:
yes, otherwise the spellcheck is unhappy with the links in the text